DeploymentCloud/Deployment.DataX/Samples/flows/eventhubbatch-product.json

```diff
@@ -0,0 +1,574 @@
+{
+	"name" : "eventhubbatch",
+	"icon" : "/img/iot.png",
+	"displayName" : "EventHub-Batch",
+	"commonProcessor" : {
+		"sparkJobConfigFolder" : "wasbs://flows@${cpConfigFolderBase}/${containerPath}/${name}",
+		"template" : {
+			"name" : "${name}",
+			"input" : {
+				"blob" : "${inputBatching}",
+				"blobSchemaFile" : "${inputSchemaFilePath}",
+				"referenceData" : "${inputReferenceData}"
+			},
+			"process" : {
+				"metric" : {
+					"eventhub" : "$keyvaultPrefix://${sparkKeyVaultName}/${metricEventHubConnectionStringKey}"
+				},
+				"timestampColumn" : "${processTimestampColumn}",
+				"watermark" : "${processWatermark}",
+				"jarUDAFs" : "${processJarUDAFs}",
+				"jarUDFs" : "${processJarUDFs}",
+				"azureFunctions" : "${processAzureFunctions}",
+				"projections" : "${processProjections}",
+				"timeWindows" : "${processTimeWindows}",
+				"transform" : "${processTransforms}",
+				"appendEventTags" : {
+				},
+				"accumulationTables" : "${processStateTables}"
+			},
+			"outputs" : "${outputs}"
+		},
+		"sparkJobTemplateRef" : "DataXBatch",
+		"jobCommonTokens" : {
+			"jobName" : "${name}",
+			"sparkJobName" : "DataXBatch-${name}",
+			"sparkJobDriverLogLevel" : "WARN",
+			"sparkJobExecutorLogLevel" : "INFO",
+			"sparkJobNumExecutors" : "${guiSparkJobNumExecutors}",
+			"sparkJobExecutorMemory" : "${guiSparkJobExecutorMemory}",
+			"sparkJobDatabricksMinWorkers" : "${guiSparkJobDatabricksMinWorkers}",
+			"sparkJobDatabricksMaxWorkers" : "${guiSparkJobDatabricksMaxWorkers}",
+			"sparkDatabricksToken" : "${guiSparkDatabricksToken}",
+			"sparkJobDatabricksAutoScale" : "${guiSparkJobDatabricksAutoScale}",
+			"processedSchemaPath" : null
+		},
+		"jobs" : [
+			{
+				"partitionJobNumber" : "1"
+			}
+		]
+	},
+	"metrics" : {
```

Comment:
we still generate those for batch?

Reply:
yes we do. It still send the metrics to eventhub and store them into the cache. We can display them later if needed.